hoomd/variant.py

```diff
@@ -47,6 +47,13 @@ def max(self):
         """The maximum value of this variant."""
         return self._max()
 
+    def __getstate__(self):
```

Comment:
pydocstyle is erroring because `__geststate__` and `__setstate__` don't have documentation.